pyston/filters/managers.py

```diff
@@ -174,6 +175,35 @@ def filter(self, resource, qs, request):
         raise NotImplementedError
 
 
+def get_flat_lookups(q):
+    if isinstance(q, Q):
+        result = []
+        for child in q.children:
+            result += get_flat_lookups(child)
+        return result
+    else:
+        return [q[0]]
```

Comment:
return list of lookups from Q objects. For example:

user__issues__name = xy and user__first_name= ab

returns

[user__issues__name, user__first_name]